### ISSUE TITLE
reading in a file with relative path does not work

### ISSUE THREAD

**Issue by Almenon** [Duroktar/Wolf#24]
The python code should be executed in the same directory as the file

```python
import os 
os.getcwd() #? outputs C:\Program Files(x86)\ Microsoft VS Code ~ not what we want

with open("testReadFile.txt") as f:   #testReadFile is in the same directory, should read fine
  a = f.read() #? FileNotFoundError: [Errno 2] No such file or directory: 'testReadFile.txt'
```

**Comment by Almenon** [comment]
well this should be enough issues to tide you over for the weekend :P

**Comment by Duroktar** [comment]
Hah, I got all the other but not this one :P Patch coming in the next few..

**Comment by Duroktar** [comment]
Fixed in v0.3.9 available now!

----

![path_example](https://user-images.githubusercontent.com/19689932/37556744-db0a232e-29d8-11e8-81f4-ffd6e7367c19.png)
